app/helpers/application_helper.rb

```diff
@@ -22,6 +22,14 @@ def default_page_header
     "CASA / Volunteer Tracking"
   end
 
+  def session_signout_link
+    if user_signed_in?
+      destroy_user_session_path
+      elsif all_casa_admin_signed_in?
```

Comment:
the linter might dislike this

Reply:
Definitely. This was mostly to get the sign out working.
There will need to be some refactor issues for this, the `active_class` helper method, the `menu_item` helper method and a bunch more. Also how the filtering works, and more :)

Reply:
Also, please pull and re-test. I didn't realize that the /public/assets director was in the gitignore file.